lms/services/jstor.py

```diff
@@ -29,16 +47,56 @@ def via_url(self, request, document_url):
 
         return via_url(
             request,
-            document_url,
+            document_url=self._get_public_url(document_url),
             content_type="pdf",
-            options={"via.jstor.site_code": self._site_code},
+            # Show content partner banner in client for JSTOR.
+            options={"via.client.contentPartner": "jstor"},
```

Comment:
To me this makes a whole lot more sense than the code we had in Via catching this all over the place